backfill/backfill_manager.py

```
@@ -0,0 +1,474 @@
+            tasks.append(task)
+        
+        # Wait for all tasks to complete
+        results = await asyncio.gather(*tasks, return_exceptions=True)
+        
+        # Check for errors
+        for result in results:
+            if isinstance(result, Exception):
+                logger.error(f"Batch processing error: {result}")
+                self.stats.errors += 1
+    
+    async def _process_file(self, file_info: Dict, year: str) -> Optional[Dict]:
```

Ruff (0.11.9)

`_process_file` is too complex (12 > 10)

(C901)